src/app/config/config.yml

```diff
@@ -26,6 +26,7 @@ framework:
         name: %zikula.session.name%
         handler_id: session.handler.legacy
         storage_id: session.storage.legacy
+        cookie_httponly: false
```

Comment:
If this is set to false, the session id cannot be retrieved in JavaScript via `document.cookie`. This is currently required however to make our AJAX requests work.